test/library/packages/Zarr/ZarrPartial.chpl

```diff
@@ -0,0 +1,59 @@
+use Zarr;
+use IO;
+use FileSystem;
+use Random;
+use BlockDist;
+
+proc test3D() {
+  const dist = new blockDist(boundingBox={0..10, 0..10, 0..10});
+  const fullD = dist.createDomain({0..10, 0..10, 0..10});
+  const partialD = dist.createDomain({0..0, 0..0, 0..10});
+
+  var ones: [fullD] real(32) = 1;
+  var twos: [fullD] real(32) = 2;
+
+  if (isDir("ones")) then rmTree("ones");
+  if (isDir("twos")) then rmTree("twos");
```

Comment:
I think you should create these under a temporary directory, i.e. something from ``mktemp``. Otherwise if this test were run from a different directory it could remove pre-existing directories named "ones" and "twos".

Reply:
Done!